.github/workflows/build.yml

```diff
@@ -8,14 +8,21 @@ on:
     branches: ["main"]
 jobs:
   build_test:
-    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - os: ubuntu-latest
```

Comment:
We're implicitly relying on github runners for `ubuntu-latest` to be x86_64 here, right?

Reply:
maybe we can check with `uname -m`, but what to do if it's not x86_64, just fail immediately ?

Reply:
Yes